src/Parser/Core/Modules/Items/IshkarsFelshieldEmitter.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Wrapper from 'common/Wrapper';
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+
+import Combatants from '../Combatants';
+import Analyzer from '../../Analyzer';
+
+class IshkarsFelshieldEmitter extends Analyzer {
+    static dependencies = {
+        combatants: Combatants,
+    };
+
+    damageAbsorbed = 0;
+    damageDealt = 0;
+
+    on_initialized() {
+        const selectedCombatant = this.combatants.selected;
+        this.active = selectedCombatant.hasTrinket(ITEMS.ISHKARS_FELSHIELD_EMITTER.id);
+    }
+
+    on_byPlayer_absorbed(event) {
+        const spellId = event.ability.guid;
+        if (spellId === SPELLS.FELSHIELD_ABSORB.id) {
+            this.damageAbsorbed += event.amount;
+        }
+    }
+
+    on_byPlayer_damage(event) {
+      const spellId = event.ability.guid;
+      if (spellId === SPELLS.FELSHIELD_DAMAGE.id) {
+        this.damageDealt += event.amount;
```

Comment:
To be consistent with the rest of the code base please also include `event.absorbed`. Absorbed damage usually is beneficial to the raid.

Reply:
I'm a bit confused though by what you're saying. The item's description is: 
Place a Felshield on an ally, absorbing X damage for 9 sec. When the shield is consumed or expires, it explodes dealing 50% of the absorbed damage as Fire split amongst all enemies within 8 yds. (1 Min Cooldown)

This means it has 2 effects, absorb and then blow up to damage enemies for half the amount absorbed. This specific part is the damage to enemies part, not the absorbed part.

Reply:
``this.damageDealt += event.amount + (event.absorbed || 0);``

Reply:
OH! Nevermind, you mean how much the boss absorbed. got it!